tests/rados/monitor_configurations.py

```diff
@@ -204,6 +204,23 @@ def set_config(self, **kwargs):
         log.info(f"Value for config: {kwargs['name']} was set")
         return True
 
+    def get_config(self, **kwargs) -> tuple:
+        """
+        Retrieve the config param in monitor config database
+        Usage - config get <section> [<name>]
+        Config parameters
+        Args:
+            **kwargs: Any other param that needs to be set
+                1. section: which section of daemons to target
+                    allowed values: global, mon, mgr, osd, mds, client
+                2. name: name of the config param for the selection
+        Returns: tuple output of ceph config get <> command
```

Comment:
please add the example of the return here, so caller functions know what to expect. ( since we are returning a tuple, not just the value requested.

Also, Please handle scenarios were the configs are not yet set. If you try to fetch those, the command would throw exception.

Reply:
Example has been added and return has been changed from tuple to string.

Even if the configs are not set, it will not raise an exception, rather return an empty string (verified manually) or the default compiled value.
Ref - https://docs.ceph.com/en/latest/rados/configuration/ceph-conf/#commands